pallets/inflation/src/lib.rs

```diff
@@ -389,27 +481,6 @@ pub mod pallet {
 
             new_inflation_config
         }
-
-        /// Check if payout cap limit would be reached after payout.
-        fn is_payout_cap_limit_exceeded(payout: Balance) -> bool {
```

Comment:
why can't we keep this safety measure?

Reply:
Please check the comment above, I'll tag you.